lib/services/ios-project-service.ts

```diff
@@ -104,10 +104,16 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 		}).future<void>()();
 	}
 
-	public createProject(frameworkDir: string, frameworkVersion: string): IFuture<void> {
+	public createProject(frameworkDir: string, frameworkVersion: string, pathToTemplate?: string): IFuture<void> {
 		return (() => {
 			this.$fs.ensureDirectoryExists(path.join(this.platformData.projectRoot, IOSProjectService.IOS_PROJECT_NAME_PLACEHOLDER)).wait();
-			if(this.$options.symlink) {
+			if(pathToTemplate) {
+				// Copy everything except the template from the runtime
+				this.$fs.readDirectory(frameworkDir).wait()
+					.filter(dirName => dirName.indexOf(IOSProjectService.IOS_PROJECT_NAME_PLACEHOLDER) === -1)
+					.forEach(dirName => shell.cp("-R", path.join(frameworkDir, dirName), this.platformData.projectRoot));
```

Comment:
Do we care that shell.cp can fail? We need to call shell.error to check for failures.